playground/bicep/BicepSample.ApiService/Program.cs

```diff
@@ -16,12 +16,12 @@
 
 builder.AddSqlServerDbContext<MyDbContext>("db");
 builder.AddNpgsqlDbContext<MyPgDbContext>("db2");
-builder.AddAzureCosmosDB("cosmos");
-builder.AddRedis("redis");
-builder.AddAzureBlobService("blob");
-builder.AddAzureTableService("table");
-builder.AddAzureQueueService("queue");
-builder.AddAzureServiceBus("sb");
+builder.AddAzureCosmosDbClient("cosmos");
+builder.AddRedisClient("redis");
+builder.AddAzureBlobServiceClient("blob");
+builder.AddAzureTableServiceClient("table");
+builder.AddAzureQueueServiceClient("queue");
```

Comment:
What do folks think about shortening these?

```suggestion
builder.AddAzureBlobsClient("blob");
builder.AddAzureTablesClient("table");
builder.AddAzureQueuesClient("queue");
```

Reply:
The underlying class names are:

* `Azure.Storage.Blobs.BlobServiceClient`
* `Azure.Data.Tables.TableServiceClient`
* `Azure.Storage.Queues.QueueServiceClient`

Is it confusing if we drop "Service" from these names? In general I like to stick with the naming of the underlying thing. But agree that @DamianEdwards's suggestion looks cleaner.

Reply:
Keep the underlying names, that way we don't need to think too much 😄

Reply:
FWIW we're adding `Client` to all the methods which happens to match the underlying types in this case but not all cases.

Reply:
I won’t die on this hill but we should optimize for less case by case naming conventions.

I could go with either.

Reply:
I think we should aim to go with something generic like `Add[LogicalName]Client`, e.g. `AddRedisClient`, `AddAzureBlobsClient`, `AddSqlClient`, etc. (obvious exception for EF Core related methods that operate on a `DbContext`).

Reply:
Lets close then.

Reply:
> Lets close then.

Is that "Just do it"?

Reply:
Done, also changed AddAzureKeyVault~Secrets~Client